ansible_mitogen/connection.py

```diff
@@ -120,10 +120,15 @@ def _connect_ssh(spec):
     """
     Return ContextService arguments for an SSH connection.
     """
-    if C.HOST_KEY_CHECKING:
+    if spec.ssh_host_key_checking() == 'no' or spec.ssh_host_key_checking() is False:
```

Comment:
Instead of potentially calling `spec.ssh_host_key_checking()` 4 times, please make this a var and then refer to it in your if statements:
```
ssh_host_key_checking = spec.ssh_host_key_checking()
if ssh_host_key_checking in ('no', False):
  check_host_keys = 'ignore'
elif ssh_host_key_checking in ('yes', True):
  check_host_keys = 'enforce'
else:
  ...
```